ecs-logs-collector.sh

```diff
@@ -362,6 +363,41 @@ get_docker_logs() {
   ok
 }
 
+get_docker_config() {
+  try "collect system's Docker configuration"
+  dstdir="${info_system}/docker_config"
```

Comment:
Nitpick: I realize that this convention doesn't exist in this repo, but I'd prefer `dstdir` vars inside of functions be declared as `local`. 

```
local dstdir="..."
```

This is truly a nitpick and probably doesn't need to be handled in this PR.